packages/ssr/jest.js

```diff
@@ -12,7 +12,14 @@ module.exports = class extends NodeEnvironment {
   setup() {
     return Promise.resolve();
   }
+
+  // Jest@21 uses `dispose` Jest@22 uses `teardown`.
+  // To support both we provide both.
+  // See the Jest@22 Changelog: https://github.com/facebook/jest/blob/master/CHANGELOG.md#features-3
   teardown() {
     return Promise.resolve();
   }
+  dispose() {
```

Comment:
Do you think it would be better to call `this.dispose()` from `teardown()`? It makes it more clear that `dispose()` is the future implementation, and if anything changes, we only need to change it there.

Reply:
Yea I think I like that. 👍

Reply:
Oh so I made this change and then realized that I think your comment is reversed. Future is `teardown` so call it from `dispose`.

Reply:
Put the logic in whatever jest 22 uses I reckon.

Reply:
👍